driver/controller_test.go

```diff
@@ -200,6 +206,10 @@ func TestControllerExpandVolume(t *testing.T) {
 		},
 		{
 			name: "requested size less than minimum supported size",
+			godo_volume: &godo.Volume{
+				ID:            "volume-id",
+				SizeGigaBytes: (minimumVolumeSizeInBytes / giB),
```

Comment:
Should we subtract a bit to make sure we're really below the minimum size (and not hitting it exactly)?